dod/src/main/java/com/revolut/rxdata/dod/DataObservableDelegate.kt

```diff
@@ -114,7 +114,7 @@ class DataObservableDelegate<Params : Any, Domain : Any> constructor(
                 ).doAfterSubscribe {
                     if (loading) {
                         subject.onNext(Data(content = memCache, loading = loading))
-                        fetchFromNetwork(params)
+                        fetchFromNetwork(memCache, params)
```

Comment:
actually `memCache` can be replaced with `null` since we will check latest memory inside anyways.